Libplanet.Net.Tests/TestUtils.cs

```diff
@@ -112,6 +119,21 @@ public static class TestUtils
                 id,
                 ImmutableArray<byte>.Empty);
 
+        public static Vote SignVote(Vote vote, PrivateKey privateKey)
```

Comment:
I think this should be included in `Vote`, like `Vote.Sign()`. We can consider making vote as `class` (was `struct`).

Reply:
[4afb5c6](https://github.com/planetarium/libplanet/pull/2003/commits/4afb5c612cee8bb7841d2b5a73fb45f6a358cf6d) Amended.

Reply:
Do we still need this test function? How about replace them with `.Sign()`?

Reply:
To be honest. it was bothering me... Eh, I'll fix it up.

Reply:
[4e61652](https://github.com/planetarium/libplanet/pull/2003/commits/4e61652f941f5be7477e835df5b456184def17a3) Amended!